content/en/docs/about/membership.md

```diff
@@ -0,0 +1,193 @@
+code review is focused on code quality and correctness, approval is focused on
+holistic acceptance of a contribution including: backwards / forwards
+compatibility, adhering to API and flag conventions, subtle performance and
+correctness issues, interactions with other parts of the system, etc.
+
+**Defined by:** *approvers* entry in an OWNERS file in a repo owned by the Kubeflow project.
+
+Approver status is scoped to a part of the codebase.
+
+### Requirements
+
+The following apply to the part of codebase for which one would be an approver in an [OWNERS] file (for repos using the bot).
+
+- Reviewer of the codebase for at least 3 months
+- Primary reviewer for at least 10 substantial PRs to the codebase
+- Reviewed or merged at least 30 PRs to the codebase
+- Nominated by a WG Lead, Chair or owner
+  - With no objections from other Leads or owners
+  - Done through PR to update the top-level OWNERS file
+
+**Note:** Working Group Leads may nominate and approve `Approvers` that don't meet these requirements due to exceptional circumstances. While acceptable in the short term, Working Group Leads should ensure that these `Approvers` eventually meet the requirements
+
+### Responsibilities
+
+The following apply to the part of codebase for which one would be an approver
+in an [OWNERS] file (for repos using the bot).
+
+- All responsibilities that reviewers have
+- Approver status may be a precondition to accepting large architectural contributions
+- Demonstrate sound technical judgement
+- Responsible for project quality control via code reviews
+  - Focus on holistic acceptance of contribution such as dependencies with other features, backwards / forwards
+    compatibility, API and flag definitions, etc
+- Expected to be responsive to review requests
+- Mentor contributors and reviewers
+
+### Privileges
+
+- All privileges that reviewers have
+- May approve code contributions for acceptance
+
+## Inactive members
+
+*Members are continuously active contributors in the community.*
+
+A core principle in maintaining a healthy community is encouraging active
+participation. It is inevitable that people's focuses will change over time and
+they are not expected to be actively contributing forever.
+
+However, being a member of one of the Kubeflow GitHub organizations comes with
+an elevated set of permissions. These capabilities should not be used by those
+that are not familiar with the current state of the Kubeflow project.
+
+Therefore members with an extended period (1 year) away from the project with no activity
+will be removed from the Kubeflow GitHub Organizations and will be required to
+go through the org membership process again after re-familiarizing themselves
+with the current state.
+
+### How inactivity is measured
+
+Inactive members are defined as members of one of the Kubeflow Organizations
+with **no** contributions across any organization within 12 months.
```

Comment:
Maybe we should explicitly say ""with no technical and non-technical contributions""

Reply:
Isn't that the same as "with no contributions"?

Reply:
I think, we can just call it contributions if we have separate section in our doc which explains: What is contribution ?

Reply:
We can follow similar guideline to what Argo project is doing: https://github.com/argoproj/argoproj/blob/main/community/membership.md#how-inactivity-is-measured

Basically DevStats should give us a sense of activity.

Reply:
Added.